### PR TITLE
🐛 enhancedHeader Fix sticky position Guard missions

### PR DESCRIPTION
**Pull request by JRH-1997**

### CODE REVIEW

File: docs/nl_NL/apps.md [hunk 1]
```diff
@@ -19,4 +19,80 @@ Zoals beschreven in de uitleg van de [Appstore](appstore.md), zijn er modules di
...
+        <td></td>
+        <td>User-ID</td>
+    </tr>
+    <tr>
+        <td>Korte gebouwinformatie</td>
+        <td></td>
+        <td>Voertuigstatus bij gebouwen</td>
+    </tr>
+</tbody>
+</table>
```
Comment (jxn-30) [inline docs/nl_NL/apps.md:97]: Why not using Markdown tables? :thinking:

Comment (JRH-1997) [reply docs/nl_NL/apps.md:97]: Because I used the same  type of tabel  as above this table in the same file. But I can change that if you want.

Comment (jxn-30) [reply docs/nl_NL/apps.md:97]: I prefer to use HTML only if it cannot be done with MD because Vue is used inside ^^

Comment (JRH-1997) [reply docs/nl_NL/apps.md:97]: 👍 I'll change that

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**